packages/Button/src/Button.styles.js

```diff
@@ -123,13 +128,7 @@ export const commonStyles = css`
   text-decoration: none;
   vertical-align: middle;
 
-  &:focus {
-    ${stylers.focusRing.bordered()}
-  }
-
   &:active {
-    box-shadow: ${tokens.highlight.active.noBorder.boxShadow}, inset 0 1px 1px 0 rgba(0, 0, 0, 0.1),
```

Comment:
This is a deliberate regression.  Getting the inset `box-shadow` to compose nicely with all the other box shadows, depending on which `kind` and state the button is, is too much complexity for a tiny, fleeting UI detail, in my strong opinion.